wurst/systems/classes/mage/elementalist/Meditate.wurst

```diff
@@ -0,0 +1,126 @@
+package Meditate
+
+import HashMap
+import ClosureEvents
+import ClosureTimers
+import AbilityObjEditing
+import Orders
+import OverCharge
+
+@configurable constant int ABILITY_ID = 'ESMD'
+@configurable constant real COOLDOWN = 60
+@configurable constant int MANACOST = 0
+@configurable constant real DURATION = 8
+@configurable constant real PERIOD_INTERVAL = 0.1
+@configurable constant real MANA_PER_SECOND = 15
+@configurable constant string SPELL_ICON = "ReplaceableTextures\\CommandButtons\\BTNDispelMagic.blp"
+@configurable constant string EFFECT_PATH = "Abilities\\Spells\\Undead\\DarkRitual\\DarkRitualTarget.mdl"
+@configurable constant string TOOLTIP_NORM = "M - Meditate"
```

Comment:
Let's try to move towards `QWER...` at hotkeys.

Reply:
All the spells will eventually use QWER, at the moment the spell locations in the spellbook are still unknown.